tests/unit/utils/test_rest_api.py

```diff
@@ -0,0 +1,17 @@
+from unittest import TestCase
+
+from openslides.utils.rest_api import ModelSerializer
+
+
+class ValidatorTest(TestCase):
+    def test_XSS_protection(self):
+        data = {
+            'String alaskjdlk': 'slwq<a><p>test<script></p>relel</a>slsl',
```

Comment:
The test strings are not consistent: Please use a strict test szenario (e. g. always 10 random chars):
```python
'String 1': 'tuveegi2Ho<a><p>tuveegi2Ho<script></p>Boovai7esu</a>ee4Yaiw0ei',
'List 1': [
    'sAa3efae1ji<script><p>iek9in6ahV</p>',
    {
        'String 1': 'oiKuaGie8v',
        'String 2': 'ohJie1Usha<script iek9in6ahV'
    }
]
```